src/common/colorspaces.c

```diff
@@ -316,15 +317,11 @@ static cmsHPROFILE _create_lcms_profile(const char *desc, const char *dmdd,
   cmsToneCurve *out_curves[3] = { trc, trc, trc };
   cmsHPROFILE profile = cmsCreateRGBProfile(whitepoint, primaries, out_curves);
 
+  if(v2) cmsSetProfileVersion(profile, 2.1);
```

Comment:
Oh I missed that. Internal ICC should always be v2, because v4 removes the white point tag at all, so the RGB primaries need to be manually adapted to D50.

Reply:
I suppose I should close this PR, right? It is wrong and the issue is not there?

Reply:
Not sure at this point. Inputing the white point is definitely a good thing anyway, it can't hurt. Since external ICC profiles work, the problem is most probably in the internal profiles we create.

Reply:
Ok, I'll close this for now.